services/src/main/java/org/keycloak/services/resources/admin/AdminRoot.java

```diff
@@ -63,6 +63,8 @@
 public class AdminRoot {
     protected static final Logger logger = Logger.getLogger(AdminRoot.class);
 
+    private static final boolean ADMIN_ENABLED = Profile.isFeatureEnabled(Profile.Feature.ADMIN_API);
```

Comment:
Sorry, this one too :)

Reply:
Other than that one, LGTM.

Reply:
Shouldn't this be:
```suggestion
    private static final boolean ADMIN_ENABLED = Profile.isFeatureEnabled(Profile.Feature.ADMIN2) || Profile.isFeatureEnabled(Profile.Feature.ADMIN);
```

Then ADMIN_ENABLED used below instead of repeating that?

Reply:
I think this needs to be controlled with a dedicated ADMIN_API flag, otherwise users couldn't configure an admin-api only keycloak instance without admin-console.

Reply:
Forgot that AdminRoot is both for UI and API ;)

So ignore this comment